app/src/main/java/org/oppia/app/walkthrough/WalkthroughActivity.kt

```diff
@@ -8,8 +8,8 @@ import javax.inject.Inject
 
 /** Activity that contains the walkthrough flow for users. */
 class WalkthroughActivity : InjectableAppCompatActivity(), WalkthroughFragmentChangeListener {
-  @Inject lateinit var walkthroughActivityPresenter: WalkthroughActivityPresenter
-
+  @Inject
+  lateinit var walkthroughActivityPresenter: WalkthroughActivityPresenter
```

Comment:
This should be in single line as per style guidelines.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting

Reply:
Got it. thanks.